test.cpp

```diff
@@ -402,6 +403,44 @@ std::map<std::string, std::function<void()>> construct_tests() {
    EXPECT(!y.defined());
  };
 
+ tests["autograd/serialization/binary"] = []() {
+   auto x = at::CPU(at::kFloat).randn({5, 5});
+   auto y = at::Tensor();
+
+   std::stringstream ss;
+   {
+     cereal::BinaryOutputArchive archive(ss);
```

Comment:
Add a test for a non-binary format?

Reply:
Can do, but for portable binary we have up use the general functions already.

Reply:
Gotcha, I didn't notice that, thanks.